docs/xdr/features/collect/integrations/cloud_and_saas/cloudflare/cloudflare-access-request.md

```diff
@@ -0,0 +1,80 @@
+uuid: 588a448b-c08d-4139-a746-b2b9f366e34b
+name: Cloudflare Access Request
+type: intake
+
+## Overview
+
+Cloudflare is a global network designed to make everything you connect to the Internet secure, private, fast, and reliable.
+
+In this documentation, you will learn how to collect and send Cloudflare Access Request logs to SEKOIA.IO.
+
+{!_shared_content/operations_center/detection/generated/suggested_rules_588a448b-c08d-4139-a746-b2b9f366e34b_do_not_edit_manually.md!}
+
+{!_shared_content/operations_center/integrations/generated/588a448b-c08d-4139-a746-b2b9f366e34b.md!}
+
+## Configuration
+
+### Create the intake on SEKOIA.IO
+
+Go to the [intake page](https://app.sekoia.io/operations/intakes) and create a new intake from the format Cloudflare.
+
+### Configure events forwarding on Cloudflare
+
+{!_shared_content/operations_center/integrations/cloudflare_necessary_info.md!}
+
+#### Create a Logpush job
+
+Configure a [Logpush job](https://developers.cloudflare.com/logs/reference/logpush-api-configuration/){:target="_blank"} with the following destination:
+
+`https://intake.sekoia.io/plain/batch?header_X-SEKOIAIO-INTAKE-KEY=<YOUR_INTAKE_KEY>`
+
+
+To do so, you can manage Logpush with cURL:
+
+```bash
+$ curl -X POST 'https://api.cloudflare.com/client/v4/accounts/<CLOUDFLARE_ACCOUNT_ID>/logpush/jobs' \
+-H 'Authorization: Bearer <CLOUDFLARE_API_TOKE?>' \
+-H "Content-Type: application/json" \
+-d '{
+    "dataset": "access_request",    
```

Comment:
Seems to be [`access_requests`](https://developers.cloudflare.com/logs/reference/log-fields/account/access_requests/)
```suggestion
    "dataset": "access_requests",    
```